src/sass/styles.scss

```diff
@@ -126,15 +126,15 @@ $border-thin: 1px solid $color-mid-light !default;
   display: inline-block;
   min-width: 8.5rem;
 
-  * {
+  & > * {
     border-right: $border-thin;
     margin: 0;
     padding-left: 0;
     padding-right: 0;
     width: 44px;
```

Comment:
In order to keep the requested `width` of 44px (minimum width required to be clickable on mobile devices), you should move this `width` to the `button` element or one of its class selectors.